db/migrate/20210216023520_drop_language_columns.rb

```diff
@@ -0,0 +1,8 @@
+class DropLanguageColumns < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured do
+      remove_column :articles, :language
+      remove_column :users, :language_settings
```

Comment:
While this also removes the index, it means this migration is non-reversible. I thought that generally, we're ok with that but let me know if you want to change it.

Reply:
Note: I believe we already have non reversible migrations in the directory